docs/user_guide.md

```diff
@@ -19,19 +19,14 @@
 6. Wait for all auto-approve triggers for all parties to deploy...
 7. Drag and drop roles to specific parties: (then click Next)
     - Assign `Bank` the `Custody`, `Distribution`, and `Settlement` roles
-    - Assign `Exchange` the `Custody`, `Trading`, `Matching`, and `Settlement` roles
+    - Assign `Exchange` the `Exchange` role
     - Assign `Ccp` the `Clearing` role
-9. Drag and drop automation to specific parties: (then click Next)
-    - Assign `Bank` the `SettlementInstructionTrigger`
-    - Assign `Exchange` the `MatchingEngine`, and `SettlementInstructionTrigger`
-    - Assign `Ccp` the `ClearingTrigger`
 9. Request services from specific parties: (then click Next)
     - As `Issuer` request `Custody`, `Issuance` services from `Bank`
-    - As `Bank` request `Custody` services from `Alice`
+    - As `Alice` request `Custody` services from `Bank`
+    - As `Bob` request `Custody` services from `Bank`
     - As `Ccp` request `Custody` services from `Bank`
-    - As `Exchange` request `Listing` services from `Exchange`
-    - As `Alice` request `Custody` services from `Exchange`
-    - As `Bob` request `Custody` services from `Exchange` and `Bank`
```

Comment:
just confirming, Eric requested that Bob ask Bank for Custody [here](https://docs.google.com/spreadsheets/d/19bVrXqWwfnKwky6IzeSuHkpbhlDA84R9HsdVIe0RuvQ/edit?pli=1#gid=1119206321) which is why we added

Reply:
that's still there, I just removed Bob requesting custody from the Exchange from that line

Reply:
merp ur right 👍